fastseq/optimizer/fairseq/beam_search_optimizer.py

```diff
@@ -62,7 +74,7 @@ def _reorder_encoder_out(self, encoder_out, new_order):
         return encoder_out
 
 
-@replace(TransformerModel)
+@replace(TransformerModel, not use_el_attn)
 class TransformerModelV2(TransformerModel):
```

Comment:
When use_el_attn is True, this V2 version won't be effect?

Reply:
This class will not be replaced here , it will be replaced in el_attn_optimizer file.